perceval/utils/postselect.py

```diff
@@ -26,19 +26,35 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+from __future__ import annotations
+
+from copy import copy
+from typing import TypeAlias, Self
 
 from .statevector import BasicState, BSDistribution, StateVector
+from exqalibur import PostSelect as ex_PostSelect
+from exqalibur import LogicOperator
+
+from perceval.utils.logging import deprecated
 
-import json
-import re
+operands: TypeAlias = LogicOperator
 
 
-class PostSelect:
+class PostSelect(ex_PostSelect):
```

Comment:
Can we drop this inheritance if we drop the methods eq, gt, etc. methods too?
That way, we'd just have a TypeAlias for PostSelect (?)